static/src/javascripts/projects/common/modules/experiments/tests/participation-star-ratings.js

```diff
@@ -66,7 +71,33 @@ define([
             },
             {
                 id: 'star-rating',
-                test: function () {}
+                test: function () {
+                    if (Object.create) {
+                        var starRatings = Object.create(lowFrictionParticipation),
+                            description = '';
+
+                        if (config.page.headline && config.page.author) {
+                            description = 'Is your rating the same as ' + config.page.author + ' on "' + config.page.headline + '"?';
```

Comment:
Strictly, shouldn't `config.page.author` be in the possessive? e.g. "Is your rating the same as Mark Kermode's on _Son of Saul_?"

You'd have to handle names ending in _s_, though:

```javascript```
function possessive(name) {
  var lastChar = name.substr(-1);
  var postfix = (lastChar === 's') ? "'" : "'s";
  return name + postfix;
}